spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/pubsub/GcpPubSubProperties.java

```diff
@@ -43,6 +47,12 @@
 	 */
 	private String emulatorHost;
 
+	/**
+	 * List of packages containing types that are whitelisted for deserialization from message
+	 * payloads.
+	 */
+	private List<String> trustedPackages = Collections.emptyList();
```

Comment:
`String[]` maybe, since that's what you need to pass to Jackson.

Reply:
done